proselint/checks/spelling/misc.py

```diff
@@ -24,7 +24,6 @@ def check(text):
     misspellings = [
         ["a lot",             ["alot"]],
         ["academically",      ["academicly"]],
```

Comment:
Since we're adding a new rule for this, we should move all pre-existing `ly` -> `ally` and related corrections to the new files

Reply:
Hi! Thank you so much for the kind words, it really means a lot. I went ahead and made the requested change, and looking through it I believe every non-miscellaneous spelling error has been moved to its proper error code. retractable was in both  misc and able_ible, so I went ahead and removed that from misc altogether. If I made any mistakes in the flow of editing a pull request, I apologize.

Reply:
No need to apologise. Thank you, and looks like you got everything. I'll check briefly then run CI and if it passes we're all clear to land your first PR